README.md

```diff
@@ -211,34 +221,69 @@ Link to other services:
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
 ```
 
+# How to get an output
+
+There are 4 possible outcomes depending on the input:
+
+1. The input is a valid payment file.
+Outcome:
+    - The transactions are logged,
+    - The payment file is moved from `inbound` to `archive`,
+    - Each valid payment request is sent to the Topic `ffc-pay-request`.
+
+2. The input is a payment file where the amount of payment requests does not match the batch header.
+Outcome: 
+    - An error message is logged,
+    - The payment file is moved from `inbound` to `quarantine`.
+
+3. The input is a payment file with an invalid batch header (e.g. wrong date format)
+Outcome: 
+    - An error message is logged,
+    - The payment file is moved from `inbound` to `quarantine`.
+
+4. The input is a payment file where the batch header value matches the payment line values but one or more payment line does not match the corresponding invoice line total.
+Outcome:
+    - The valid payment requests are logged and error messages are logged for each invalid request,
+    - The payment file is moved from `inbound` to `archive`,
+    - Each valid payment request is sent to Topic `ffc-pay-request`,
+    - An invalid event is sent to `ffc-pay-event` for each invalid payment request.
+
+# How to stop the Batch Processor
+
+Run the command `docker-compose down`
+
+# How to test the Batch Processor
 ## Test structure
 
 The tests have been structured into subfolders of `./test` as per the
 [Microservice test approach and repository structure](https://eaflood.atlassian.net/wiki/spaces/FPS/pages/1845396477/Microservice+test+approach+and+repository+structure)
 
 ### Running tests
+Tests can be run in several modes
+- [Run and exit](#run-and-exit)
+- [Run tests with file watch](#run-tests-with-file-watch)
 
 A convenience script is provided to run automated tests in a containerised
 environment. This will rebuild images before running tests via docker-compose,
 using a combination of `docker-compose.yaml` and `docker-compose.test.yaml`.
 The command given to `docker-compose run` may be customised by passing
 arguments to the test script.
 
-Examples:
 
-```
+
+
 # Run all tests
-scripts/test
+`scripts/test`
 
 # Run tests with file watch
-scripts/test -w
-```
+`scripts/test -w`
```

Comment:
There's also a debug tests option which is `scripts/test -d`.

Essentially it's the same as watch but attaches a debugger too so tests can be debugged.